open-craft/xblock-group-project-v2
# MCKIN-24383 xblock internationalization

In order to make a Group project v2  translatable, I have integrated translation and added a minimal number of hooks to Python code and templates. By this PR, the text should be translated into the end user’s language.

Testing instructions:
-  checkout to this branch
- and to verify the strings are now getting translated for all languages.
